Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+
+class TransformStack: NSObject {
+    
+    static var shared: TransformStack = TransformStack()
+    
+    weak var transformDelegate : TransformDelegate!
```

Comment:
For delegate, I think usually we use `optional` over `force unwrapping`
I suggest to change it to `weak var transformDelegate: TransformDelegate?`

Reply:
changes made